chart/templates/server-ide-configmap.yaml

```diff
@@ -104,6 +104,34 @@ options:
 
 defaultIde: "code"
 defaultDesktopIde: "code-desktop"
+
+clients:
+  vscode:
+    defaultDesktopIDE: "code-desktop"
+    desktopIDEs: ["code-desktop"]
+    installationSteps: [
```

Comment:
**suggestion**: Probably not the best line to comment, but going with the same width as the workspace-related element above sounds good! What do you think?